### DISCUSSION TITLE
Malicious mobile apps spread via GitHub pages

### DISCUSSION THREAD

**Discussion by affiliateseries** [community/community#136650] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi Team,
I came to know that GitHub pages is being used for spreading malicious bank's mobile apps. I have already created a ticket 2957106
Best
Aff

**Comment by ebndev** [comment]
Hi @affiliateseries, 👋🏻 We really appreciate you flagging this and opening a support ticket.

The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
